Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -1189,14 +1185,18 @@ static void webkitWebViewGetProperty(GObject* object, guint propId, GValue* valu
     case PROP_DEFAULT_CONTENT_SECURITY_POLICY:
         g_value_set_string(value, webkit_web_view_get_default_content_security_policy(webView));
         break;
-#if PLATFORM(WPE)
     case PROP_THEME_COLOR: {
+#if PLATFORM(WPE)
         auto* color = static_cast<WebKitColor*>(fastMalloc(sizeof(WebKitColor)));
         webkit_web_view_get_theme_color(webView, color);
         g_value_take_boxed(value, static_cast<gconstpointer>(color));
+#else
+        GdkRGBA color;
+        webkit_web_view_get_theme_color(webView, &color);
+        g_value_set_boxed(value, static_cast<gconstpointer>(&color));
+#endif
```

Comment:
Very unimportant nit: it's semantically nicer to guard the GTK code with `PLATFORM(GTK)` guards, since theoretical future ports could use the WebKitColor code, but GdkRGBA will only ever be useful for GTK port.

Reply:
Sure thing, I'll change this to use `PLATFORM(GTK)` before landing 👌🏼